bohr/heuristics/bug.py

```diff
@@ -0,0 +1,365 @@
+# from bohr.heuristics.templates
+from bohr.snorkel_utils import BUG, BUGLESS, ABSTAIN, commit_lf, keyword_lfs
+import re
+from bohr.snorkel_utils import Commit
+
+BUG_MESSAGE_KEYWORDS = [
+   'bad',
+   ['bug', 'bugg'],
+   'broken',
+   'concurr',
+   ['correct', 'correctli'],
+   'corrupt',
+   'crash', 
+   ['deadlock', 'dead lock'],
+   'defect',
+   'endless',
+   'ensur',
+   'error',
+   ['fail', 'failur'],
```

Comment:
could you please remind me why we need to group some keywords, wouldn't be it in the same as having two different labeling functions with 'fail' and 'failur' keywords?